### PR TITLE
Add ability to check for LEADERSHIP_COMPANY_LABEL to display leadership modal on portal

### PR DESCRIPTION
**Pull request by B77Mills**
- Add env variable for LEADERSHIP_COMPANY_LABEL
- Check leadershipCompanyLable in company labels
 -- Check that leadershipEnable is true && if a leadershipCompayLabel is set. 
-- If so check if the label is in the list of company labels.
 -- If not set the LeadershipEnable to false.

### CODE REVIEW

**Comment by B77Mills** [comment]
@solocommand Is this okay to be merged?

File: services/graphql/src/resolvers/config.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const {
   CONTACT_URL,
   CONTACT_TEXT,
   LEADERSHIP_ENABLED,
+  LEADERSHIP_COMPANY_LABEL,
```
Comment (solocommand) [inline services/graphql/src/resolvers/config.js:6]: This new variable needs to be defined in the envalid configuration using the `str` validator. When doing so, please confirm the behavior when the label isn't defined -- it should be nulled out in the graphql response when not present.

File: services/graphql/src/definitions/config.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ module.exports = gql`
     contactUrl: String
     contactText: String
     leadershipEnabled: Boolean!
+    leadershipCompanyLabel: String!
```
Comment (solocommand) [inline services/graphql/src/definitions/config.js:14]:
```suggestion
    leadershipCompanyLabel: String
```
This label shouldn't be required

File: services/app/app/routes/portal.js [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
     const variables = { input: { hash } };
     const model = await this.apollo.query({ query, variables, fetchPolicy: 'network-only' }, 'contentHash');
     if (!model) throw new Error('Invalid URL');
+    
+    // Check that leadershipEnable is true && if a leadershipCompay Label is set.  
+    // If so check if the label is in the list of company labels.
+    // If not set the LeadershipEnable to false.
+    if (this.config.leadershipEnabled && this.config.leadershipCompanyLabel) {
+      if(!model.labels.includes(this.config.leadershipCompanyLabel)) this.config.leadershipEnabled = false;
+    }
```
Comment (solocommand) [inline services/app/app/routes/portal.js:20]: Make this a computed property instead using the same fields. You can watch against the flag and label from the config and the model's labels to net the same result without the side effect of changing the config. You can then use that computed property in the template directly.

Something like 
```js
  leadershipEnabled: computed('config.{leadershipEnabled,leadershipCompanyLabel}', 'model.labels.[]', function() {
    // ex.; return config && label && model.labels.includes(label)
  }),
```

Just make sure to account for it being enabled without a required label as well :)

**Review by solocommand — CHANGES_REQUESTED** [review]
Looks good with a couple of changes.

File: services/graphql/src/resolvers/config.js (hunk 1 above)
Comment (B77Mills) [reply services/graphql/src/resolvers/config.js:6]: @solocommand wouldn't this return an empty string if it was setup in the env.js as a `str` or am i missing something?

**Review by solocommand — APPROVED** [review]

**Pull request merged**